CHANGES.md

```diff
@@ -11,12 +11,29 @@
 
 ##### Additions :tada:
 
-- Added `ArcGISTiledElevationTerrainProvider.fromUrl`, `CesiumTerrainProvider.fromUrl`, `GoogleEarthEnterpriseMetadata.fromUrl`, `GoogleEarthEnterpriseTerrainProvider.fromMetadata`, `VRTheWorldTerrainProvider.fromUrl`, and `createWorldTerrainAsync` for better async flow and error handling.
+- Added `ArcGisMapServerImageryProvider.fromUrl`, `ArcGISTiledElevationTerrainProvider.fromUrl`, `BingMapsImageryProvider.fromUrl`, `CesiumTerrainProvider.fromUrl`, `GoogleEarthEnterpriseMetadata.fromUrl`, `GoogleEarthEnterpriseImageryProvider.fromMetadata`, `GoogleEarthEnterpriseTerrainProvider.fromMetadata`, `IonImageryProvider.fromAssetId`, `SingleTileImageryProvider.fromUrl`, `TileMapServiceImageryProvider.fromUrl`, `VRTheWorldTerrainProvider.fromUrl`, and `createWorldTerrainAsync` for better async flow and error handling.
```

Comment:
Please note that I kept the deprecation strings as 1.102/1.104. This matches the existing ready deprecation messages that are already in place, so it will be easier to find/replace them as a whole before https://github.com/CesiumGS/cesium/pull/11059 is merged, most likely for 1.103.